### PR TITLE
feat: Adds RabbitMQ

### PR DESCRIPTION
**Pull request by hofmeister**

### CODE REVIEW

File: templates/kapeta/block-type-service/router.go.hbs [hunk 1]
```diff
@@ -5,14 +5,13 @@ import (
...
 )
 
-func RegisterRouters(e *echo.Echo, cfg sdkgoconfig.ConfigProvider) {
-{{#providers-of-type 'kapeta/resource-type-rest-api'}}
+func RegisterRouters(e *echo.Echo, cfg kapeta.ConfigProvider) {
     {{#kaplang-rest-methods spec.source namespace=metadata.name}}
         rest.Create{{controller-name this}}Router(e, cfg);
     {{/kaplang-rest-methods}}
     rest.CreateHealth(e, cfg);
-{{/providers-of-type}}
```
Comment (sorenmat) [inline templates/kapeta/block-type-service/router.go.hbs:17]: can you remove this??? shouldn't it close the one on line 11

File: src/target/template-helpers.ts [hunk 2]
```diff
@@ -305,7 +305,7 @@ export const addTemplateHelpers = (engine: HandleBarsType, data: any, context: a
                     }
 
                     let out =  `var ${valueName} ${value.type.name}\n`
-                    out += `if ${valueName}, err = request.GetQueryParam(ctx, "${value.name}", &${valueName}); err != nil {\n`
+                    out += `if _, err = request.GetQueryParam(ctx, "${value.name}", &${valueName}); err != nil {\n`
```
Comment (sorenmat) [inline src/target/template-helpers.ts:308]: If we are doing this, then we should only return the error I guess

Comment (hofmeister) [reply src/target/template-helpers.ts:308]: It was just a bit confusing to both have the return value and the reference parameter ? Also it didn't work

File: templates/kapeta/block-type-service/router.go.hbs (hunk 1 above)
Comment (hofmeister) [reply templates/kapeta/block-type-service/router.go.hbs:17]: Nope I couldn't :) But changed it a bit - health only exists once and should always be there

**Pull request merged**